src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -130,93 +129,18 @@ export const Accept: FC<AcceptProps & StripeProps> = props => {
         return
       }
 
-      if (orderOrError.error.code !== "payment_requires_action") {
+      if (orderOrError.error.code) {
         handleAcceptError(orderOrError?.error)
         return
       }
 
-      // TODO: Remove below fixedOrderOrError logic and fixFailedPayment once Exchange mutation is updated
-      const fixedOrderOrError = (
-        await fixFailedPayment({
-          input: {
-            creditCardId: order.creditCardId!,
-            offerId: order.lastOffer?.internalID,
-            orderId: order.internalID,
-          },
-        })
-      ).commerceFixFailedPayment?.orderOrError!
-
-      if (fixedOrderOrError.error) {
-        handleAcceptError(orderOrError?.error)
-        return
-      }
-
-      const scaResult = await stripe.handleCardAction(
-        fixedOrderOrError.actionData?.clientSecret!
-      )
-
-      if (scaResult.error) {
-        return dialog.showErrorDialog({
-          title: "An error occurred",
-          message: scaResult.error.message,
-        })
-      }
-
       onSubmit()
     } catch (error) {
       logger.error(error)
       dialog.showErrorDialog()
```

Comment:
**suggestion (non-blocking):** This is out-of-scope, but it would be sweet to drop a `trackEvent` call, similar to the one on line 107. If you' rather not, I can open a separate PR for that too 👍

Reply:
[27ead8a](https://github.com/artsy/force/pull/12869/commits/27ead8a9e1021703fc8f7154d97d7f20b594de99)